esignet-service/internal/engine/mosip/model.go

```
@@ -192,3 +192,20 @@ type ServiceError struct {
+	ResponseTime string                      `json:"responsetime,omitempty"` // usually ISO8601 / RFC3339
+	Response     *GetAllCertificatesResponse `json:"response,omitempty"`
+	Errors       []Error                     `json:"errors,omitempty"`
+}
+
+type GetAllCertificatesResponse struct {
+	AllCertificates []KycSigningCertificateData `json:"allCertificates"`
+}
+
+type KycSigningCertificateData struct {
+	KeyId           string `json:"keyId"`
+	CertificateData string `json:"certificateData"` //X509 certificate
```

golangci-lint (2.12.2)

[error] commentFormatting: put a space between `//` and comment text

(gocritic)